components/NavigationBar.js

```diff
@@ -0,0 +1,124 @@
+import Link from "next/link";
+import styled from "styled-components";
+import React from "react";
+import { useRouter } from "next/router";
+
+const StyledFooter = styled.footer`
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  height: 60px;
+  background-color: white;
+`;
+const NavigationList = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: space-around;
+`;
+const IconListItem = styled.li`
+  background-color: ${(props) => props.bgColor || ""};
```

Comment:
When passing props to a styled component, you should prefix them with "$", otherwise you will get a warning in the console.

```suggestion
  background-color: ${(props) => props.$bgColor || ""};
```